src/utils/httpInterceptor.ts

```diff
@@ -0,0 +1,209 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2021
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+import { AmtMode, Args, Connection, ConnectionType } from '../models/models'
+import Common from './common'
+
+export const HttpInterceptorAuthentications: {[key: string]: string} = {}
+
+export class HttpInterceptor {
```

Comment:
not sure this is needed anymore --- can we use HttpHandler?

Reply:
I dont believe httphandler will be a replacement. I tried parsing the messages with httpZ.parse(this.amt.acc) as HttpZResponseModel and wasn't able to get any messages to parse without throwing an exception.